test/freenet/support/MultiValueTableTest.java

```diff
@@ -250,8 +249,8 @@ public void testGetArray() {
 		Object[][] sampleObjects = 
 			createSampleKeyMultiVal(sampleKeyNumber,sampleMaxValueNumber,sampleIsRandom);
 		methodMVTable = fillMultiValueTable(sampleObjects);
-		for(int i=0;i<sampleObjects.length;i++)
-			assertTrue(Arrays.equals(((List<Object>)(sampleObjects[i][1])).toArray(),methodMVTable.getArray(sampleObjects[i][0])));
+		for (Object[] sampleObject : sampleObjects)
```

Comment:
Cast `sampleObjects`, change the type of `sampleObject`, remove cast in line below. And braces, please.